pkg/healthz/healthz_suite_test.go

```diff
@@ -1,13 +1,11 @@
 package healthz
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"kubevirt.io/client-go/testutils"
 
 	"testing"
 )
 
 func TestHealthz(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Healthz Suite")
+	testutils.KubeVirtTestSuiteSetup(t, "Hooks Suite")
```

Comment:
`"Hooks Suite"` Is this correct?

Reply:
No, fixed. Good catch. On bazel it gets replaces with something like `//pkg/healthz/healthz_suite_test` to make it easier to discover the failure path. If run with pure go it will be picked up. I wonder if this "redundancy" can somehow be removed in a follow-up.